lightning/src/ln/reorg_tests.rs

```diff
@@ -281,8 +281,9 @@ fn do_test_unconf_chan(reload_node: bool, reorg_after_reload: bool, use_funding_
 	if !reorg_after_reload {
 		if use_funding_unconfirmed {
 			let relevant_txids = nodes[0].node.get_relevant_txids();
-			assert_eq!(&relevant_txids[..], &[chan.3.txid()]);
-			nodes[0].node.transaction_unconfirmed(&relevant_txids[0]);
+			let txid = relevant_txids[0].0;
```

Comment:
Would we rather have the assertion fail if there are no relevant txids? Or is panicking fine?

Reply:
I think we'd would have also panicked before, if `relevant_txids` were empty, in which case we now panic, too? So essentially no difference there, but I now threw in a length assertion for good measure.

Reply:
Prior, we knew `relevant_txids` had length 1 since it would have been equal to `[chan.3.txid()]` in the earlier assertion.

Reply:
Yes, you are absolutely correct.